packages/desktop/src/app.js

```diff
@@ -32,7 +32,7 @@ const App = () => {
   const { actions } = useAppScope();
 
   React.useEffect(() => {
-    if (authStatus !== "authenticated") return;
+    if (user == null) return;
```

Comment:
This makes it so the the initial page load requests `/users/me` and `/ready` happen in serial instead of parallel. Without this two requests both trigger a refresh when the access token expire, which results in two parallell refresher cascades with a bunch of fetch calls.